help/Makefile

```diff
@@ -0,0 +1,130 @@
+# Makefile for Sphinx documentation
```

Comment:
What is the purpose of this sphinx docs (help folder)?

Reply:
I dontk know yet ...
but the pb_tools didnt work without it